src/main/groovy/com/stripe/TransactionServices.groovy

```diff
@@ -45,6 +45,8 @@ class TransactionServices {
         def chargeId = ec.context.chargeId
         def amount = ec.context.amount
 
+        amount = amount.toInteger() * 100
```

Comment:
Can you add a comment explaining why we need to multiply this by 100. People new to this code would benefit from knowing that stripe records USD amounts by the smallest division (cents).

Reply:
I added the comment as you requested, please check